app/src/main/java/com/example/androidthings/assistant/AssistantActivity.java

```diff
@@ -192,9 +206,9 @@ public void onConversationFinished() {
                         Log.i(TAG, "assistant conversation finished");
                         if (mLed != null) {
                             try {
-                                mLed.setValue(false);
+                                mLed.setValue(true);
```

Comment:
Shouldn't we turn off the LED in this case?

Reply:
Since the LED is configured as ACTIVE_LOW, true is actually off (I said that as a note on the description) I find it confusing, but I did not want to change the original behaviour.

Reply:
The `ACTIVE_LOW` / `ACTIVE_HIGH` are set so that setting the value to true/false does what is expected. In my case, the LEDs are `ACTIVE_LOW`, but that's where the value should be changed.

Reply:
In my case led.setValue(false) does turn it on, I'm not sure if that is a configuration problem, a hardware difference or something that changed on the OS since this project was originally made

Reply:
Actually the blinking is following this pattern, first on, then off, so at the end of each blink, it stays off. Let me know if you want me to revert all of them and change the active mode.

Reply:
I would revert the changes so that `true` => `on` and `false` => `off`.

In my experience, the LED is `ACTIVE_LOW`, and I don't know if I want to change that if that's how the VoiceHat really is, but we should probably change it there.

Reply:
It would be helpful if someone else could verify the LED state on different AIY voice kits and Android Things versions. Until then, I'm not inclined to change anything.

IMHO opening the LED with the correct mode should be a responsibility of the voiceHAT driver.

Reply:
Actually, after one more test / reading of the code, onButtonEvent the LED is off when pressed and turns on when released (pressed == false)

The code is mLed.setValue(pressed)

And that is code I have not touched.

Unless it does not work as I described for you, then we clearly have some inconsistency on hardware / operating system.

Reply:
`mLed.setValue(pressed);` makes sense using the true/on setup.

I do agree that the LED should be set up through the VoiceHat. Until that change is made and published to the driver, we should refrain from making this workaround.